torch/testing/_internal/common_optimizers.py

```diff
@@ -1543,6 +1558,13 @@ def _get_optim_inputs_including_global_cliquey_kwargs(
                 "test_tensor_lr",
                 active_if=sys.version_info < (3, 9) and sys.version_info > (3, 7),
             ),
+            DecorateInfo(
+                unittest.expectedFailure,  # Fails if use_closure
+                "CompiledOptimizerParityTests",
+                "test_correctness",
+                active_if=lambda kwargs: sys.platform == "darwin"
```

Comment:
could you open an issue to track how/why this fails?

Reply:
(and then link it here)

Reply:
done